build_scripts/release_scripts/Dockerfile

```diff
@@ -1,108 +0,0 @@
-# See docker-build.sh for usage
```

Comment:
Well that's because my first draft only had the Ubuntu Dockerfile, which I wrote from scratch. Until I was told that we need to keep the Oracle Linux one again, so I brought it back to life.